scripts/CustomMaterials.zs

```diff
@@ -10,44 +10,91 @@ import mods.gregtech.recipe.Utils;
 //MATERIALS 8000-10000 RESERVED FOR NUCLEAR-RELATED ITEMS
 //MATERIALS 10000-20000 RESERVED FOR FLUIDS
 //MATERIALS 20000-30000 RESERVED FOR SOLIDS
-//MATERIALS 29000-30000 RESERVED FOR ELEMENTS
-//MATERIALS 31000-32727 RESERVED FOR ISOTOPES
+//MATERIALS 30000-31000 RESERVED FOR ORES
+//MATERIALS 31000-32000 RESERVED FOR ISOTOPES
+//MATERIALS 32000+ RESERVED FOR ELEMENTS
 
 //----------------------------------ELEMENTS---------------------------------
 
-MaterialBuilder(29000, "iodine")
+MaterialBuilder(32000, "mystery")
+.dust()
+.color(0x141414)
+.build();
+
+<material:mystery>.setFormula("?", false);
+
+MaterialBuilder(32001, "iodine")
 .element("Iodine")
 .dust()
 .color(0x390b6e)
 .build();
 
-MaterialBuilder(29001, "hafnium")
+MaterialBuilder(32002, "hafnium")
 .element("Hafnium")
 .dust()
 .ingot()
 .color(0x1c1b18)
 .build();
 
-MaterialBuilder(29002, "chromium")
+MaterialBuilder(32003, "chromium")
 .element("Chromium")
 .dust()
 .ingot()
 .color(0xccb8c7)
 .build();
 
-MaterialBuilder(29003, "strontium")
+MaterialBuilder(32004, "strontium")
 .element("Strontium")
 .dust()
 .ingot()
 .color(0xdbdaab)
 .build();
```

Comment:
All of these lines should be removed, as the materials already exist without properties. They need properties for ingot, dust, etc to be added to them using the addProperty() method on the materials.